brigade/static/scss/style.scss

```diff
@@ -1,9 +1,6 @@
 // Import the variables first, so they can be customized.
 @import '~@codeforamerica/style/_sass/core/_variables.scss';
```

Comment:
^ This should become `@import 'core/variables'` then, because the variables must be overridden before the `main.scss` file is loaded.

Reply:
Oh of course, good catch, I'm sleeping today 😴